utl/vw-hyperopt.py

```diff
@@ -31,6 +31,7 @@ def read_arguments():
     parser = argparse.ArgumentParser()
 
     parser.add_argument('--searcher', type=str, default='tpe', choices=['tpe', 'rand'])
+    parser.add_argument('--features', type=str, help="namespaces to be used while tuning hyper params. E.g.: '--keep a --keep b'")
```

Comment:
The meaning of `features` is very overloaded in general. Should this be called `additional_cmd`, `namespace_args`, or something along those lines instead of `features`?

Reply:
Agree. Renamed the argument from _features_ to _additional_cmd_ to make it more generic.